plugins/qtosgrave/osgrenderitem.cpp

```diff
@@ -278,8 +279,10 @@ void KinBodyItem::Load()
 //        }
 
         _veclinks.push_back(LinkNodes(posglinkroot, posglinktrans));
+        _vecgeoms.at(_veclinks.size() - 1).resize(0);
 
         for(size_t igeom = 0; igeom < porlink->GetGeometries().size(); ++igeom) {
+            _vecgeoms[_veclinks.size() - 1].push_back( GeomNodes(new osg::Group(), new osg::MatrixTransform()) );
```

Comment:
@Puttichai the way you are indexing `_vecgoems` looks dangerous, perhaps there is a better way?!